client/app/components/ServicesTemplate/index.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+// import PropTypes from 'prop-types';
+import cn from 'classnames';
+import Card from 'components/SharedComponents/Card';
+import ServicesCardContent from './ServicesCardContent';
+import json from './data.json';
+
+// Styles
+import styles from './styles.scss';
+
+const ServicesTemplate = () => {
+  const filledCardsTop = json.data.topCardContent.map(({ title, subtitle, content, url, img, id }) =>
+    <Card
+      key={id}
+      cardSize="is-6"
+      {...{ title, subtitle, content, url, img }}
```

Comment:
No subtitles defined, not sure if expected or not so making subtitle optional prop